Core.lua

```diff
@@ -337,9 +345,36 @@ function ParseClassMask(bits)
     return parse
 end
 
+function ParseArmorMask(bits)
+    -- 0 == All classes can wear
+    if bits == 0 then
+        return "ANY_ARMOR"
+    end
+    local map = { 
+        PLATE = 0x001 + 0x002 + 0x020,
+        LEATHER = 0x008 + 0x200 + 0x800 + 0x400,
+        CLOTH = 0x010 + 0x080 + 0x100,
+        MAIL = 0x004 + 0x040 + 0x1000
+    }
+    local parse = bits .. ": "
+    for char, mask in pairs(map) do
+        if (BitAND(bits, mask) > 0) then
+            return char .. ""
+        end
+    end
+    -- hmm, weird case, better itemize
+    local parse = bits .. ": "
+    for char, mask in pairs(map) do
+        if (BitAND(bits, mask) == mask) then
+            parse = parse .. char .. " "
+        end
+    end
+    return parse .. ""
+end
+
 function SetCollector:ExportSetData()
```

Comment:
In a future update, let's get the export code moved into its own lua file

Reply:
> In a future update, let's get the export code moved into its own lua file

Yeah, I wanted you to have the updated set info, and didn't have enough time to actually be a good developer, hehe.  I kinda meant for that code to not be included :/